app/controllers/follows_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class FollowsController < ApplicationController
+  def create
+    current_user.follow(params[:user_id])
+    redirect_to '/users'
```

Comment:
```suggestion
    redirect_to users_path
```

のようにヘルパーメソッドを使った方が、未定義のルーティングの存在（つまりコーディングミス）に気付きやすいです。